plugin/storage/es/spanstore/writer.go

```diff
@@ -72,7 +72,7 @@ func NewSpanWriter(p SpanWriterParams) *SpanWriter {
 		client: p.Client,
 		logger: p.Logger,
 		writerMetrics: spanWriterMetrics{
-			indexCreate: storageMetrics.NewWriteMetrics(p.MetricsFactory, "index_create"),
+			indexCreate: spanstoremetrics.NewWriterDecorator(p.MetricsFactory, "index_create"),
```

Comment:
is this an example where decorator actually IS used by the storage implementation?